static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js

```diff
@@ -53,16 +57,38 @@ class FollowCardList extends Component<
         }));
     };
 
+    expandableConsentsButtonText = (): string => {
+        const joinWithOr = arr =>
+            arr.reduce(
+                (acc, val, idx, src) =>
+                    idx === 0
+                        ? val
+                        : [
+                              acc,
+                              idx + 1 >= src.length ? ' or ' : ', ',
+                              val,
+                          ].join(''),
+                ''
+            );
+        const buttonWords = [...this.state.expandableConsents]
+            .splice(0, 2)
+            .map(c => c.consent.name);
+        if (this.state.expandableConsents.length > buttonWords.length) {
+            buttonWords.push('More');
```

Comment:
this copy `or More?` reads a little strangely to me - am I the only one?

Reply:
I made this up on the spot frankly, lowercase m is a good initial fix but maybe we don't even need the "or more" bit. How do we feel about the final "or" vs doing all commas? The mockups are all commas but I read it was a typo

Reply:
We can also go with a solution that displays static text but I really like feeding the button automatically because if we ever suggest content or w/e automatically then the button will always say the truth